test_utils/tasks.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tasks used in tests
```

Comment:
They are specific to the tests, so they shouldn't be installed with celeryutils, which is why they don't live in celeryutils.  I didn't put them in `/tests`, because the project structure seems designed to use that for actual test code. 

Also, the file is also used for tasks that test the logging decorator, once that PR comes in.

Note the /tests directory doesn't have a `__init__.py` file, and the tox file explicitly adds one for the purposes of running pylint against the tests, but removes it again at the end, which makes it less suitable for code that might get imported from different places.  @jmbowman might be able to shed light on the rationale for that decision.